### PR TITLE
[droid/ios/mac] added incoming call feature to Conference sample

### PR DESCRIPTION
**Pull request by kkarasevich**
resized-image-Promo
resized-image-Promo (1)
resized-image-Promo (2)

image

### CODE REVIEW

File: Conference/Droid/DT.Samples.Agora.Conference.Droid.csproj [hunk 1]
```diff
@@ -85,7 +85,7 @@
       <Version>13.0.1-beta2</Version>
     </PackageReference>
     <PackageReference Include="Xamarin.Agora.Rtm.Android">
-      <Version>1.1.0</Version>
+      <Version>1.4.2-beta1</Version>
```
Comment (sbokatuk) [inline Conference/Droid/DT.Samples.Agora.Conference.Droid.csproj:88]: 1.4.2 version, please fix

File: Conference/iOS/DT.Samples.Agora.Conference.iOS.csproj [hunk 2]
```diff
@@ -180,14 +176,14 @@
...
     <Reference Include="System.Numerics" />
-    <Reference Include="Xamarin.Agora.Full.iOS">
-      <HintPath>..\..\packages\Xamarin.Agora.Full.iOS.3.3.0\lib\Xamarin.iOS10\Xamarin.Agora.Full.iOS.dll</HintPath>
-    </Reference>
     <Reference Include="Newtonsoft.Json">
-      <HintPath>..\..\packages\Newtonsoft.Json.13.0.1-beta2\lib\netstandard2.0\Newtonsoft.Json.dll</HintPath>
+      <HintPath>..\packages\Newtonsoft.Json.13.0.1\lib\netstandard2.0\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="Xamarin.Agora.Full.iOS">
+      <HintPath>..\packages\Xamarin.Agora.Full.iOS.3.4.0\lib\Xamarin.iOS10\Xamarin.Agora.Full.iOS.dll</HintPath>
```
Comment (sbokatuk) [inline Conference/iOS/DT.Samples.Agora.Conference.iOS.csproj:183]: RTC should be latest, 3.4.1

File: RTM/Droid/DT.Samples.Agora.Rtm.Droid.csproj [hunk 3]
```diff
@@ -199,6 +196,9 @@
     <Reference Include="Xamarin.Android.Support.Exif">
       <HintPath>..\packages\Xamarin.Android.Support.Exif.28.0.0.3\lib\monoandroid90\Xamarin.Android.Support.Exif.dll</HintPath>
     </Reference>
+    <Reference Include="Xamarin.Agora.Rtm.Android">
+      <HintPath>..\packages\Xamarin.Agora.Rtm.Android.1.4.2-beta1\lib\MonoAndroid\Xamarin.Agora.Rtm.Android.dll</HintPath>
```
Comment (sbokatuk) [inline RTM/Droid/DT.Samples.Agora.Rtm.Droid.csproj:200]: same, 1.4.2

File: RTM/Droid/Activity/MessageActivity.cs [hunk 4]
```diff
@@ -363,23 +363,23 @@ protected override void OnActivityResult(int requestCode, [GeneratedEnum] Result
...
-
-                                if(_isPeerToPeerMode)
-                                {
-                                    SendPeerMessage(rtmImageMessage);
-                                }
-                                else
-                                {
-                                    SendChannelMessage(rtmImageMessage);
-                                }
+                                //var rtmImageMessage = message as RtmImageMessage;
```
Comment (sbokatuk) [inline RTM/Droid/Activity/MessageActivity.cs:366]: why we removing working feature? do we met some problems on android?

File: Shared/packages.config [hunk 5]
```diff
@@ -41,6 +41,6 @@
   <package id="System.ValueTuple" version="4.5.0" targetFramework="portable45-net45+win8+wpa81" />
   <package id="System.Xml.ReaderWriter" version="4.3.1" targetFramework="portable45-net45+win8+wpa81" />
   <package id="System.Xml.XDocument" version="4.3.0" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Xam.Plugins.Settings" version="4.1.0-beta" targetFramework="portable45-net45+win8+wpa81" />
+  <package id="Xam.Plugins.Settings" version="3.1.1" targetFramework="portable45-net45+win8+wpa81" />
```
Comment (sbokatuk) [inline Shared/packages.config:44]: must be 4.1.0-beta

**Review by sbokatuk — CHANGES_REQUESTED** [review]
please fix packages versions, and why image support removed from android RTM?

File: Shared/packages.config (hunk 5 above)
Comment (kkarasevich) [reply Shared/packages.config:44]: > must be 4.1.0-beta

we have runtime error with this version & other projects use 3.1.1

**Pull request merged**